Web/starter_project_vanguard/components/example/Button.tsx

```diff
@@ -33,7 +35,7 @@ export const Button = ({
       {...props}
       className={clsx(
         shape === 'square' ? 'rounded-md' : 'rounded-full',
-        'bg-blue-700 text-white p-8',
+        `bg-blue-700 text-white p-8`,
```

Comment:
[nit] why the need to change the quotation mark here ? 
if you are not using dynamic content in it I would recommend sticking to `''`

Reply:
and I also don't understand why the addition of spaces above (might be fine but I don't understand the purpose)